libs/api/domains/auth-admin/src/lib/scope/models/scope-environment.model.ts

```diff
@@ -1,20 +1,64 @@
-import { Field, ObjectType } from '@nestjs/graphql'
+import { Field, ID, ObjectType } from '@nestjs/graphql'
 
 import { Environment } from '@island.is/shared/types'
-
 import { TranslatedValue } from '../../models/translated-value.model'
 
 @ObjectType('AuthAdminScopeEnvironment')
 export class ScopeEnvironment {
   @Field(() => Environment)
   environment!: Environment
 
-  @Field(() => String, { nullable: false })
+  @Field(() => Boolean)
+  enabled!: boolean
+
+  @Field(() => ID)
   name!: string
 
   @Field(() => [TranslatedValue], { nullable: false })
   displayName!: TranslatedValue[]
 
   @Field(() => [TranslatedValue], { nullable: false })
   description!: TranslatedValue[]
+
+  @Field(() => String)
+  domainName!: string
+
+  @Field(() => Number)
+  order?: number
+
+  @Field(() => String)
+  groupId?: string
+
+  @Field(() => Boolean)
+  showInDiscoveryDocument!: boolean
+
+  @Field(() => Boolean)
+  required!: boolean
+
+  @Field(() => Boolean)
+  emphasize!: boolean
+
+  @Field(() => Boolean)
+  grantToAuthenticatedUser!: boolean
+
+  @Field(() => Boolean)
+  grantToLegalGuardians!: boolean
+
+  @Field(() => Boolean)
+  grantToProcuringHolders!: boolean
+
+  @Field(() => Boolean)
+  grantToPersonalRepresentatives!: boolean
+
+  @Field(() => Boolean)
+  allowExplicitDelegationGrant!: boolean
+
+  @Field(() => Boolean)
+  automaticDelegationGrant!: boolean
+
+  @Field(() => Boolean)
+  alsoForDelegatedUser!: boolean
+
+  @Field(() => Boolean)
+  isAccessControlled?: boolean
```

Comment:
Should always be set and available as the default value would just be `false`.
```suggestion
  isAccessControlled!: boolean
```